sdk/keyvault/keyvault-admin/src/index.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export * from "./core/keyVaultClient";
+export * from "./core/keyVaultClientContext";
```

Comment:
We have an issue logged to rename core to generated in other keyvault projects. Should we perhaps do it at the very beginning for this package?